metricbeat/module/coredns/stats/_meta/fields.yml

```diff
@@ -0,0 +1,90 @@
+- name: stats
+  type: group
+  description: >
+    Contains statistics related to the coreDNS service
+  release: beta
+  fields:
+    - name: panic.count.total
+      type: long
+      description: >
+         Total number of panics
+    - name: dns.request.count.total
+      type: long
+      description: >
+         Total query count
+    - name: dns.request.duration.sec.bucket.*
+      type: object
+      object_type: double
+      description: >
+         Request duration histogram buckets in nanoseconds
+    - name: dns.request.duration.sec.sum
+      type: long
+      format: duration
+      description: >
+         Requests duration, sum of durations in nanoseconds
+    - name: dns.request.duration.sec.count
+      type: long
+      description: >
+         Requests duration, number of requests
+    - name: dns.request.size.bytes.bucket.*
```

Comment:
I wonder if split bytes make sense or if we should make this type `long` which would round the values.

Reply:
:+1:

Reply:
Sorry for joining this discussion so late, I have just seen it :grimacing:

Keeping doubles could be fine as the source uses doubles for this, changing to long makes sense because they are actually integers, but I wonder if we can overflow in some of these metrics.